js/NavigationLogoView.js

```diff
@@ -97,6 +97,11 @@ class NavigationLogoView extends Backbone.View {
     if (event.code && !['Space', 'Enter'].includes(event.code)) return;
     if (event?.preventDefault) event.preventDefault();
 
+    if(redirectToId === '/' || redirectToId === '\\' || redirectToId === 'menu' || redirectToId === 'home') {
```

Comment:
Could you make it only one option, 4 is unnecessary, of `@home`. The `@` is to match other relative paths in adapt and to distinguish from valid IDs.

Reply:
So you're suggesting just to use `@home` ?

Reply:
I'm in two minds. `@course` is a valid relative relative path. You can see that by running this in the console of a vanilla course:

```js
require('core/js/data').findById('c-200').findRelativeModel('@course')
```

Is it alway the course menu you want to go back to? Because `@home` would be a new relative path.

Reply:
I'm happy with `@course` if there is precedent for using it in other extensions.

Reply:
I don't think we need to introduce a new variable. I think they would always want to go the main menu. If (in <=1% of courses) they need to go to a submenu they could just use ID.

I'll update to `@course` now.